client-apis/decrypt.md

```diff
@@ -205,7 +201,7 @@ This operation MUST attempt to deserialize all consumable encrypted message byte
 successfully deserialized a valid [message header](../data-format/message-header.md).
 
 This operation MUST wait if it doesn't have enough consumable encrypted message bytes to
-deserialize the next field of the message header until enough consumable input bytes become or
+deserialize the next field of the message header until enough consumable input bytes become available or
```

Comment:
```suggestion
deserialize the next field of the message header until enough input bytes become consumable or
```
avoiding "available" and just using "consumable" because that word is defined in streaming.md.